### DISCUSSION TITLE
How to do LGE (Latest Github Exploit)

### DISCUSSION THREAD

**Discussion by helloIlovegimkit** [3kh0/ext-remover#1028] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
How this works is like the one with the old Github Exploit on google

How can I do this:

You can do it by clicking on the add user button then if your prompted without your school's custom sign-in then after you type in your school email click sign in options and click Github
the first cat doesn't work so you need to go to terms then -> Github docs cat then click quick-start guide --> click repos click about repository then find the word enterprise and click the blue link and then click 
[github.com/new](https://github.com/new) and click the cat on the top and search for Google (if your school has network blocking then it'll kick you out if you visit a blocked link) then click the hyperlink for Google and boom google

**Comment by helloIlovegimkit** [comment]
By the way you cannot sign into Github or else the [Github.com/new](https://Github.com/new) link doesn't work

**Comment by myschoolisass** [comment]
isnt it faster to just click terms, scroll down to find github.com, then search googleyay and go to one of the varying links?
plus, most schools no longer have sign-in options
i'm not quite sure, so you can disregard this for now

**Reply by helloIlovegimkit** [reply]
I found that you can just click docs and then do the new repo steps from there

**Comment by Averagedude69420** [comment]
a faster way is to click forgot password on the first github screen then press the cat and search up google
